public/assets/sass/pages/get-involved/council.scss

```diff
@@ -28,3 +28,67 @@ section {
     padding: 8px;
   }
 }
+
+.mailing-list-form {
+  max-width: var(--content-width);
+  margin: 40px auto;
+  padding: 5px;
+  background-color: #f0f0f0;
```

Comment:
can we use the CSS colour variables for this? I think there's a `var(--background-colour-3)` that would work well here!

Reply:
I changed it to `var(--background-colour-3)`.